db/migrate/20250730182737_add_is_neighborhood_migrated_to_devlogs.rb

```diff
@@ -0,0 +1,5 @@
+class AddIsNeighborhoodMigratedToDevlogs < ActiveRecord::Migration[8.0]
+  def change
+    add_column :devlogs, :is_neighborhood_migrated, :boolean, default: false, null: false
```

Comment:
personally I might call it `from_neighborhood` because then it automatically gives you `devlog.from_neighborhood?`, but your code will still function as-is